source/_components/cloudflare.markdown

```diff
@@ -0,0 +1,53 @@
+---
+layout: page
+title: "Cloudflare"
+description: "Automatically update your Cloudflare DNS records."
+date: 2018-07-10 20:08
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: cloudflare.png
+ha_category: Utility
+featured: false
+ha_release: 0.75
+---
+
+With the Cloudflare component, you can keep your Cloudflare records up to date.
+Cloudflare offers free Dynamic DNS service among other services.
+
+To use the component in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+cloudflare:
+  email: user@example.com
+  api_key: c2547eb745079dac9320b638f5e225cf483cc5cfdda41
+  zone: example.com
+  records:
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

Reply:
My comment is no longer valid.